views/home.njk

```diff
@@ -12,20 +12,41 @@
         <div class="usa-width-one-third">
           <p>This is a U.S. government service. Your use indicates your consent to monitoring, recording, and no expectation of privacy. Misuse is subject to criminal and civil penalties. <a href="/system-use">Read more details.</a><br></p>
           <div class="hero-heading">
-            <a href="/login">
-              <button class="usa-button">
-                {% if isUAA %}
-                  Agree and continue with cloud.gov authentication
-                  {% else %}
-                  <img src="/images/github-logo.png"> Agree and continue with GitHub
+            {% if authUAA %}
+              <div class="well-gray-lightest">
+                {% if authGithub %}
+                  <h5>Did we invite to use your cloud.gov account?</h5>
                 {% endif %}
```

Comment:
```suggestion
```

Reply:
maybe leave this as the happy path and only have the disclaimer with the github auth below?

Reply:
👍